common/src/main/java/com/wynntils/screens/base/WynntilsListScreen.java

```diff
@@ -34,7 +35,7 @@ public abstract class WynntilsListScreen<E, B extends WynntilsButton> extends Wy
     protected List<E> elements = new ArrayList<>();
 
     private final List<B> elementButtons = new ArrayList<>();
-    protected final QuestBookSearchWidget searchWidget;
```

Comment:
Is this still needed? I thought WynntilsGuideScreen can handle this.

Reply:
I don't really see what you mean to be honest. This allows to keep all the pre existing logic in WynntilsListScreen for the search bar, and properly overwrite the default search widget.

Reply:
@kristofbolyai I am also not sure what you mean here...